notebooks/MAST/TESS/beginner_astroquery_dv/beginner_astroquery_dv.ipynb

```diff
@@ -0,0 +1,301 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def parse_manifest(manifest):\n",
+    "    \"\"\"\n",
+    "    Parse manifest and add back columns that are useful for TESS DV exploration.\n",
+    "    \"\"\"\n",
+    "    results = deepcopy(manifest)\n",
+    "    filenames = []\n",
+    "    sector_range = []\n",
+    "    exts = []\n",
+    "    for i,f in enumerate(manifest['Local Path']):\n",
+    "        file_parts = np.array(np.unique(f.split(sep = '-')))\n",
+    "        sectors = list( map ( lambda x: x[0:2] == 's0', file_parts))\n",
+    "        s1 = file_parts[sectors][0]\n",
+    "        try:\n",
+    "            s2 = file_parts[sectors][1]\n",
+    "        except:\n",
+    "            s2 = s1\n",
+    "        sector_range.append(\"%s-%s\" % (s1,s2))\n",
+    "        path_parts = np.array(f.split(sep = '/'))\n",
+    "        filenames.append(path_parts[-1])\n",
+    "        exts.append(path_parts[-1][-8:])\n",
+    "\n",
+    "    results.add_column(table.Column(name = \"filename\", data = filenames))\n",
+    "    results.add_column(table.Column(name = \"sectors\", data = sector_range))\n",
+    "    results.add_column(table.Column(name = \"fileType\", data = exts))\n",
+    "    results.add_column(table.Column(name = \"index\", data = np.arange(0,len(manifest))))\n",
+    "    \n",
+    "    return results\n",
+    "\n",
+    "#Run parser and print\n",
+    "results = parse_manifest(manifest)\n",
+    "print(results['index','sectors','fileType'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Plot the DVT File\n",
+    "The time series data use to find each transit signal (or Threshold Crossing event (TCE)) is found in the dvt.fits files.  As you can see there is a dvt file. If we want the file with the most data, we should pick the one with the longest sector range in the file name.  In this case it is s0001-s0013. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(results['index', 'sectors', 'fileType'][results['sectors'] == \"s0001-s0013\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plot the entire DV median detrended, time series\n",
+    "The median detrended fluxes are stored in the first extension under 'LC_DETREND'. Notice in the plot before that L98-59, while in the continuous viewing zone, was not observed during every sector."
```

Comment:
"Notice in the plot before" what plot are you referring to? If you mean the one after this cell, remove the word "before," if there is supposed to have already been a plot, it seems like it got deleted somewhere along the way.